generative_ai/test_gemini_examples.py

```diff
@@ -37,6 +39,33 @@ def test_gemini_guide_example() -> None:
     assert "scones" in text
 
 
+def test_gemini_pro_basic_example() -> None:
+    text = gemini_pro_basic_example.generate_text(PROJECT_ID, LOCATION)
+    text = text.lower()
+    assert len(text) > 0
+    assert "recipe" in text or "ingredients" in text or "table" in text
+
+
+def test_gemini_pro_config_example() -> None:
+    import urllib.request
+
+    #  download the image
+    fname = "scones.jpg"
+    url = "https://storage.googleapis.com/generativeai-downloads/images/scones.jpg"
+    urllib.request.urlretrieve(url, fname)
+
+    if os.path.isfile(fname):
+        text = gemini_pro_config_example.generate_text(PROJECT_ID, LOCATION)
+        text = text.lower()
+        assert len(text) > 0
+        assert "recipe" in text or "table" in text
```

Comment:
suggestion: the sample is asking to describe the picture (prompt: "what is in the image"), so probably the text will mention scones, cookies, coffee etc.